layouts/partials/tabs.html

```diff
@@ -15,7 +15,7 @@ <h3 class="subtitle is-5 is-muted">{{ $subtitle }}</h3>
         <nav class="cd-tabs__navigation">
         <ul class="cd-tabs__list">
             <li>
-            <a href="#tab-inbox" class="cd-tabs__item cd-tabs__item--selected">
+            <a href="#tab-visualisation" class="cd-tabs__item cd-tabs__item--selected">
```

Comment:
The visualisation tab id was named tab-inbox and I renamed it to tab-visualisation so that it can be referred to in other tabs content (if required).  There are only two places where this id 'tab-inbox' was used in sources - both within the same file. No other content is referring to the id tab-inbox in html/md. For now, I will revert it back to the id name 'tab-inbox' and check out locally what's going on - at least I did not see tab content blanking out.

Reply:
Ah! Got it - renaming to tab-vis is working now - no blanking out.  The link was opening reference in a new window - that wasn't needed. Thanks for catching this.

Reply:
Fixed, tested https://github.com/numpy/numpy.org/commit/7f6ee54b18e82595dd448ec5b73f03f597e914c5